library/std/src/os/fuchsia/mod.rs

```diff
@@ -3,4 +3,6 @@
 #![stable(feature = "raw_ext", since = "1.1.0")]
 
 pub mod fs;
+#[unstable(feature = "fuchsia_exit_status_aborted", issue = "none")]
+pub mod process;
```

Comment:
Looks like you changed to `fuchsia_exit_status` in `process.rs`, but not here or the test

Reply:
You're quick! I just caught that as well and already updated it.